pkg/commands/run.go

```diff
@@ -133,6 +133,9 @@ func initFlagSet(fs *pflag.FlagSet, cfg *config.Config, m *lintersdb.Manager, is
 	fs.StringVar(&lsc.Errcheck.Exclude, "errcheck.exclude", "",
 		"Path to a file containing a list of functions to exclude from checking")
 	hideFlag("errcheck.exclude")
+	fs.StringSliceVar(&lsc.Errcheck.ExcludeFunctions, "errcheck.exclude-functions", nil,
+		"List of functions to exclude from checking")
+	hideFlag("errcheck.exclude-functions")
```

Comment:
as you can see the comment on the top of this section, the flags are "deprecated", could you remove those lines?